tests/pkunit1_test.py

```diff
@@ -10,6 +10,17 @@ def test_case_dirs():
     from pykern import pkunit
 
     for d in pkunit.case_dirs():
+        if d.basename == 'xlsx':
+            return
         i = d.join('in.txt').read()
         pkunit.pkeq(d.basename + '\n', i)
         d.join('out.txt').write(i)
+
+def test_xlsx_to_csv_conversion():
```

Comment:
This doesn't need to use case_dirs, just file_eq. Then you wouldn't need the special case. Move this into pkunit_test.py.